scipy/special/special/gamma.h

```diff
@@ -10,19 +10,14 @@ SPECFUN_HOST_DEVICE T gamma(T x) {
     return cephes::Gamma(x);
 }
 
-template <typename T>
-std::complex<T> gamma(std::complex<T> z) {
-    return specfun::cgama(z, 1);
-}
-
 template <typename T>
 SPECFUN_HOST_DEVICE T gammaln(T x) {
     return cephes::lgam(x);
 }
 
 template <typename T>
-std::complex<T> gammaln(std::complex<T> z) {
-    return specfun::cgama(z, 0);
+SPECFUN_HOST_DEVICE inline std::complex<T> gamma(std::complex<T> z) {
```

Comment:
`template` functions are also `inline` implicitly, so `inline` is not needed here. The rules are: template functions AND class / struct member functions are implicit inline. Other functions not.